src/unit_tests/wazuh_db/test_wdb_agent.c

```diff
@@ -1055,8 +1055,8 @@ void test_wdb_update_agent_version_error_socket(void **state)
     expect_value(__wrap_cJSON_AddStringToObject, string, "nodename");
     expect_string(__wrap_cJSON_AddStringToObject, name, "agent_ip");
     expect_value(__wrap_cJSON_AddStringToObject, string, "agentip");
-    expect_string(__wrap_cJSON_AddNumberToObject, name, "sync_status");
-    expect_value(__wrap_cJSON_AddNumberToObject, number, 1);
+    expect_string(__wrap_cJSON_AddStringToObject, name, "sync_status");
+    expect_value(__wrap_cJSON_AddStringToObject, string, "syncreq");
```

Comment:
We should use `expect_string` instead of `expect_value`

Reply:
Fixed